<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlWindow</class>
 <widget class="QMainWindow" name="ControlWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ControlWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="MainWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>521</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QTabWidget" name="tabWidget">
         <property name="maximumSize">
          <size>
           <width>782</width>
           <height>503</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="usesScrollButtons">
          <bool>false</bool>
         </property>
         <property name="documentMode">
          <bool>true</bool>
         </property>
         <property name="tabsClosable">
          <bool>false</bool>
         </property>
         <property name="movable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="Stats">
          <property name="maximumSize">
           <size>
            <width>761</width>
            <height>468</height>
           </size>
          </property>
          <attribute name="title">
           <string>Statistical info</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QGroupBox" name="GPSGroupBox">
             <property name="title">
              <string>GPS coordinates</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <widget class="GraphWidget" name="GPSGLWidget" native="true"/>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QGroupBox" name="VelocitiesGroupBox">
             <property name="title">
              <string>Velocities</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="GraphWidget" name="VelocitiesGLWidget" native="true"/>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QGroupBox" name="CameraGroupBox">
             <property name="title">
              <string>Camera</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="0" column="0" alignment="Qt::AlignHCenter">
               <widget class="QRadioButton" name="tpvButton">
                <property name="text">
                 <string>3rd person view</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0" alignment="Qt::AlignHCenter">
               <widget class="QRadioButton" name="fpvButton">
                <property name="text">
                 <string>1st person view</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QGroupBox" name="AnglesGroupBox">
             <property name="title">
              <string>Gyroscope angles</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0">
               <widget class="GraphWidget" name="AnglesGLWidget" native="true"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="Options">
          <property name="maximumSize">
           <size>
            <width>761</width>
            <height>468</height>
           </size>
          </property>
          <attribute name="title">
           <string>Options</string>
          </attribute>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0" colspan="2">
            <layout class="QGridLayout" name="grid">
             <item row="0" column="0">
              <widget class="QGroupBox" name="groupBox_7">
               <property name="title">
                <string>Forces</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_14">
                <item row="0" column="0">
                 <layout class="QGridLayout" name="gridLayout_18">
                  <item row="0" column="1">
                   <widget class="QDoubleSpinBox" name="doubleSpinBox_11">
                    <property name="minimum">
                     <double>0.010000000000000</double>
                    </property>
                    <property name="singleStep">
                     <double>0.010000000000000</double>
                    </property>
                    <property name="value">
                     <double>9.810000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QDoubleSpinBox" name="doubleSpinBox_12">
                    <property name="maximum">
                     <double>0.990000000000000</double>
                    </property>
                    <property name="singleStep">
                     <double>0.010000000000000</double>
                    </property>
                    <property name="value">
                     <double>0.020000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_18">
                    <property name="text">
                     <string>Resistance coefficient:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_17">
                    <property name="text">
                     <string>Gravitational constant:</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="1" rowspan="2">
              <widget class="QGroupBox" name="gps">
               <property name="title">
                <string>GPS</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_8">
                <item row="0" column="0">
                 <widget class="QCheckBox" name="gpsNoiseCheckBox">
                  <property name="text">
                   <string>GPS noise</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="gpsLabel">
                  <property name="text">
                   <string>Parameters:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <layout class="QHBoxLayout" name="gpsMathExpect">
                  <item>
                   <widget class="QLabel" name="gpsMathExpectLabel">
                    <property name="text">
                     <string>Math expectation:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="gpsMathExpectSpinBox"/>
                  </item>
                 </layout>
                </item>
                <item row="3" column="0">
                 <layout class="QHBoxLayout" name="gpsStdLayout">
                  <item>
                   <widget class="QLabel" name="stdLabel">
                    <property name="text">
                     <string>Standart deviation:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="stdSpinBox"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="2" rowspan="2">
              <widget class="QGroupBox" name="groupBox_5">
               <property name="title">
                <string>Angle</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_11">
                <item row="0" column="0">
                 <widget class="QCheckBox" name="checkBox_6">
                  <property name="text">
                   <string>Gyroscope noise</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Parameters:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                   <widget class="QLabel" name="label_11">
                    <property name="text">
                     <string>Math expectation:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="doubleSpinBox_7"/>
                  </item>
                 </layout>
                </item>
                <item row="3" column="0">
                 <layout class="QHBoxLayout" name="horizontalLayout_6">
                  <item>
                   <widget class="QLabel" name="label_12">
                    <property name="text">
                     <string>Standart deviation:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="doubleSpinBox_8"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="0" rowspan="2">
              <widget class="QGroupBox" name="groupBox_8">
               <property name="title">
                <string>Physics options</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_15">
                <item row="0" column="0">
                 <layout class="QGridLayout" name="gridLayout_13">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_19">
                    <property name="text">
                     <string>Physics FPS:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QDoubleSpinBox" name="doubleSpinBox_13">
                    <property name="minimum">
                     <double>1.000000000000000</double>
                    </property>
                    <property name="maximum">
                     <double>100.000000000000000</double>
                    </property>
                    <property name="singleStep">
                     <double>1.000000000000000</double>
                    </property>
                    <property name="value">
                     <double>30.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_20">
                    <property name="text">
                     <string>Mass:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QDoubleSpinBox" name="doubleSpinBox_14">
                    <property name="minimum">
                     <double>0.100000000000000</double>
                    </property>
                    <property name="maximum">
                     <double>10.000000000000000</double>
                    </property>
                    <property name="singleStep">
                     <double>0.100000000000000</double>
                    </property>
                    <property name="value">
                     <double>1.500000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="label_21">
                    <property name="text">
                     <string>Troque limit:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QDoubleSpinBox" name="doubleSpinBox_15">
                    <property name="minimum">
                     <double>2.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0">
                   <widget class="QLabel" name="label_22">
                    <property name="text">
                     <string>Prob of action:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1">
                   <widget class="QDoubleSpinBox" name="doubleSpinBox_16">
                    <property name="maximum">
                     <double>1.000000000000000</double>
                    </property>
                    <property name="singleStep">
                     <double>0.010000000000000</double>
                    </property>
                    <property name="value">
                     <double>0.750000000000000</double>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item row="2" column="1" rowspan="2">
              <widget class="QGroupBox" name="groupBox_6">
               <property name="title">
                <string>Velocity</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_12">
                <item row="0" column="0">
                 <widget class="QCheckBox" name="checkBox_7">
                  <property name="text">
                   <string>Velocity noise</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_13">
                  <property name="text">
                   <string>Parameters:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <item>
                   <widget class="QLabel" name="label_14">
                    <property name="text">
                     <string>Math expectation:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="doubleSpinBox_9"/>
                  </item>
                 </layout>
                </item>
                <item row="3" column="0">
                 <layout class="QHBoxLayout" name="horizontalLayout_8">
                  <item>
                   <widget class="QLabel" name="label_15">
                    <property name="text">
                     <string>Standart deviation:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="doubleSpinBox_10"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item row="2" column="2" rowspan="2">
              <widget class="QGroupBox" name="camera">
               <property name="title">
                <string>Camera</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_10">
                <item row="0" column="0">
                 <widget class="QCheckBox" name="cameraCheckBox">
                  <property name="text">
                   <string>Camera noise</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="cameraLabel">
                  <property name="text">
                   <string>Parameters:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <layout class="QHBoxLayout" name="cameraMathExpectLayout">
                  <item>
                   <widget class="QLabel" name="cameraMathExpectLabel">
                    <property name="text">
                     <string>Math expectation:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="cameraMathExpectSpinBox"/>
                  </item>
                 </layout>
                </item>
                <item row="3" column="0">
                 <layout class="QHBoxLayout" name="cameraStdLayout">
                  <item>
                   <widget class="QLabel" name="cameraStdLabel">
                    <property name="text">
                     <string>Standart deviation:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="cameraStdSpinBox"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item row="3" column="0" rowspan="2">
              <widget class="QGroupBox" name="groupBox_9">
               <property name="title">
                <string>Camera options</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_16">
                <item row="0" column="0">
                 <layout class="QGridLayout" name="gridLayout_17">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>View angle:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QSpinBox" name="spinBox">
                    <property name="minimum">
                     <number>10</number>
                    </property>
                    <property name="maximum">
                     <number>179</number>
                    </property>
                    <property name="value">
                     <number>60</number>
                    </property>
                    <property name="displayIntegerBase">
                     <number>10</number>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_16">
                    <property name="text">
                     <string>View distance:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QSpinBox" name="spinBox_2">
                    <property name="minimum">
                     <number>100</number>
                    </property>
                    <property name="maximum">
                     <number>1000</number>
                    </property>
                    <property name="value">
                     <number>200</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QGroupBox" name="delays">
               <property name="title">
                <string>Delays</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_9">
                <item row="0" column="0">
                 <layout class="QFormLayout" name="layout">
                  <item row="0" column="0">
                   <widget class="QLabel" name="readOutLabel">
                    <property name="text">
                     <string>Readout time delay:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QDoubleSpinBox" name="readoutSpinBox"/>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="execTimeLabel">
                    <property name="text">
                     <string>Execution time delay:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QDoubleSpinBox" name="execTimeSpinBox"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item row="4" column="2">
              <layout class="QVBoxLayout" name="buttons">
               <item>
                <widget class="QPushButton" name="save">
                 <property name="text">
                  <string>Save</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="reset">
                 <property name="text">
                  <string>Reset</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>815</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSimulation">
    <property name="title">
     <string>Simulation</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="actionSave_Data"/>
   </widget>
   <addaction name="menuSimulation"/>
   <addaction name="menuData"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSave_Data">
   <property name="text">
    <string>Save Data</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GraphWidget</class>
   <extends>QWidget</extends>
   <header>graphwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
